projects/js-packages/charts/tsconfig.json

```diff
@@ -5,5 +5,5 @@
 	},
 	// List all sources and source-containing subdirs.
 	"include": [ "./src" ],
-	"exclude": [ "node_modules", "dist", "**/stories/**" ]
+	"exclude": [ "node_modules", "dist" ]
```

Comment:
Fixes TS complaints about missing `React` import.

![Screenshot 2025-07-04 at 4 07 07 PM](https://github.com/user-attachments/assets/3429556e-5a01-4361-b4d5-2aacc06ce02a)

Why This Works
- The base TypeScript config at tools/js-tools/tsconfig.base.json has "jsx": "react-jsx"
- React 17+ supports the new JSX transform that doesn't require import React
- Stories files now inherit these settings and work correctly